src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -229,6 +231,17 @@ public void Initialize(bool generateNewDatabaseId = false)
                      : DocumentDatabase.Configuration.Core.DataDirectory.FullPath));
             }
 
+            if (DocumentDatabase.Configuration.Core.RunInMemory == false)
+            {
+                string disableMarkerPath = DocumentDatabase.Configuration.Core.DataDirectory.Combine("disable.marker").FullPath;
+                if (File.Exists(disableMarkerPath))
+                {
+                    throw new DatabaseDisabledException("Unable to open database: " + _name + ", it has been manually disabled via the file: '" + disableMarkerPath +"'." +
```

Comment:
better to use interpolation and wrap the name in `''` maybe?

Reply:
Want to double-check that... after removing this file, how the database can be loaded again?

Reply:
Enable / disable, on my tries.

Reply:
![image](https://user-images.githubusercontent.com/116915/184096268-eccfa022-76bf-4ecf-a384-74ebac3a7970.png)

And then you remove the file and click here:

![image](https://user-images.githubusercontent.com/116915/184096455-9918b1c6-90ec-4bdb-b35b-90ebeb1c4894.png)


And you have:

![image](https://user-images.githubusercontent.com/116915/184096484-ae7dd160-769c-4459-8a69-89c56282f900.png)